apps/opik-backend/src/main/java/com/comet/opik/OpikApplication.java

```diff
@@ -74,6 +75,8 @@ public void run(OpikConfiguration configuration, Environment environment) {
         // Resources
         var jersey = environment.jersey();
 
+        CORS.registerFilterIfEnabled(configuration, environment);
```

Comment:
I would move this setup to a module. Using [this](https://xvik.github.io/dropwizard-guicey/4.0.1/guide/module-autowiring/), you can probably achieve what you need to configure.

Reply:
Fixed in https://github.com/comet-ml/opik/pull/498/commits/0839a7db91f5512a2e83c20250ef0ea1af14d2df